### PR TITLE
Add the features of an optional header and an optional footer text

### PR DESCRIPTION
**Pull request by s7dhansh**

### CODE REVIEW

File: examples/collage-with-text.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    "./img/src2.jpg", // source can be a relative file path
+    "./img/src3.jpg", // source can be a relative file path
+    "./img/src4.jpg", // source can be a relative file path
+    "./img/src5.jpg", // source can be a relative file path
+  ],
+  width: 3, // number of images per row
+  height: 2, // number of images per column
+  imageWidth: 350, // width of each image
+  imageHeight: 250, // height of each image
+  // backgroundColor: "#cccccc", // optional, defaults to #eeeeee.
```
Comment (peterkhayes) [inline examples/collage-with-text.js]: Could you either uncomment this line or remove the slashes?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+## Disclaimer
+Forked from https://github.com/classdojo/photo-collage (`npm i photo-collage`)
+
```
Comment (peterkhayes) [inline README.md]: If you want me to merge, please remove this message :)

**Comment by peterkhayes** [comment]
Added a couple of comments; after that LGTM

**Comment by s7dhansh** [comment]
Done

**Pull request merged**